src/utils/utils_common.h

```diff
@@ -40,6 +40,12 @@ typedef enum umf_purge_advise_t {
 #define IS_NOT_ALIGNED(value, align)                                           \
     ((align != 0 && (((value) & ((align)-1)) != 0)))
 #define ALIGN_UP(value, align) (((value) + (align)-1) & ~((align)-1))
+#ifndef ALIGN_UP_SAFE
```

Comment:
This ifndef looks suspicious. Do we need it. If yes why?

Reply:
It was needed because of the doubled definition of this macro. I've removed the other definition now. Thanks for catching this one, less rubbish.

Reply:
We don't need that, my omission, now removed.